backend/battle/tests/test_view.py

```diff
@@ -74,3 +81,178 @@ def test_verify_if_not_returns_winner_missing_pokemons_teams(self):
 
         with self.assertRaises(IndexError):
             get_winner_for(self.team_creator, self.team_opponent)
+
+
+class ListBattlesTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.battle = baker.make('battle.Battle')
+        self.user = User.objects.create(email='example@example.com')
+        self.user.set_password('admin')
+        self.user.save()
+
+    def test_login_user_can_acess_battle_list(self):
+        self.client.login(username=self.user.email, password='admin')
+        response = self.client.get(reverse('battle_list'))
+        self.assertEqual(response.status_code, 200)
+
+    def test_data_returns_empty_list(self):
```

Comment:
The naming could be improved here, I don't think `data` is a good fit here

Reply:
yup, I fixed this on the other PR, sorry because there is on both PRs

Reply:
I'm going to fix it on this PR as well